tests/.pylintrc

```diff
@@ -277,10 +278,10 @@ exclude-too-few-public-methods=
 ignored-parents=
 
 # Maximum number of arguments for function / method.
-max-args=5
+max-args=10
```

Comment:
I don't think that it is a good idea to make it so large by default, people always can disable it manually...

Reply:
When I brought this up in the PPT issue (https://github.com/lincc-frameworks/python-project-template/issues/256), this is the one I thought might be most controversial =D . If you'd like to return it to `5`, we could re-open that issue (and I'd prefer to make the change upstream).